### DISCUSSION TITLE
Wren API

### DISCUSSION THREAD

**Discussion by matthieu-perso** [Canner/WrenAI#557] [Q&A]
Hi everyone,

I'd like to use Wren AI without the UI and interact with it through an API. Is there any easy way to do this now ?

**Comment by cyyeh** [comment]
> Hi everyone,
> 
> I'd like to use Wren AI without the UI and interact with it through an API. Is there any easy way to do this now ?

@matthieu-perso Welcome to join our discord server: https://discord.gg/5DvshJqG8Z

How Wren AI API works
https://discord.com/channels/1227143286951514152/1249992463977021511
